src/test/java/model/LottoNumberTest.java

```diff
@@ -0,0 +1,71 @@
+package model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LottoNumberTest {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final String LOTTO_NUMBER_MUST_BE_POSITIVE = "로또 번호는 양수여야 합니다.";
+    private static final String LOTTO_NUMBER_MUST_BETWEEN_ONE_AND_FORTY_FIVE = "로또 번호는 1과 45 사이의 숫자여야 합니다.";
+
+    @DisplayName("1부터 45 사이의 랜덤한 번호 생성을 테스트 한다.")
+    @Test
+    void generateRandomNumberTest() {
+        // given
+
+        // when
+        int number = LottoNumber.generateRandomNumber();
+
+        // then
+        assertThat(number).isBetween(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER);
+    }
+
+    @DisplayName("음수로 LottoNumber 생성시 예외를 테스트 한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {-1, -8, -17, -20})
+    void validateLottoNumberIsPositiveTest(int number) {
+        // given
+
+        // when
+
+        // then
+        assertThatThrownBy(() -> new LottoNumber(number))
```

Comment:
저도 given when then 구조 꼭 만들어서 테스트하는 입장으로 구조를 지켜주신 점에서는 마음의 안정을 느꼈습니다. 
소소한 팁이지만, 예외 테스트의 경우 when 절이 조금 애매해지는 경우가 있습니다. 
저 같은 경우 이런 경우 아래와 같이 작성합니다.
```suggestion
        // when // then
        assertThatThrownBy(() -> new LottoNumber(number))
```

추가적으로 ParameterizedTest 를 하는 경우 given 상황이 미리 주어지는 경우가 잦기 때문에 이런 경우에는 //given을 명시하지 않고있습니다. 

(이는 컨벤션 같은게 아니라 제 개인적인 코딩 스타일 이라는 점! 참고만 해주세요)

Reply:
좋은 팁 감사합니다 :) 참고하도록 하겠습니다.